src/components/ImportRecordsFlow/stepOne/index.js

```diff
@@ -19,36 +16,61 @@ export default function StepOne(props) {
         setFieldOptions(getSchemaFieldOptions(schemaFields));
     }, [schemaFields]);
 
-    return (
-        <StyledContainer>
-            <StyledVisualPicker
-                label="Select the option type that do you want to do"
-                onChange={onChangeAction}
-                value={actionOption}
-            >
-                <VisualPickerOption
+    const onChange = event => {
+        onChangeAction(event.target.value);
+    };
+
+    const options = [
+        {
+            value: 'add-records',
+            label: (
+                <OptionPicker
                     name="add-records"
-                    footer={<VisualPickerOptionFooter label="Add new records" />}
-                >
-                    <AddRecordsIcon />
-                </VisualPickerOption>
-                <VisualPickerOption
+                    value={actionOption}
+                    label="Add New records"
+                    description="Allows adding all records without checking for duplicates in existing records."
+                />
+            ),
+            name: 'add-records',
+            id: 'add-records',
+        },
+        {
+            value: 'merge-records',
+            label: (
+                <OptionPicker
                     name="merge-records"
-                    footer={<VisualPickerOptionFooter label="Merge exist records" />}
-                >
-                    <MergeRecordsIcon />
-                </VisualPickerOption>
-            </StyledVisualPicker>
-            <RenderIf isTrue={isMergeOption}>
-                <Select
-                    label="Match Field"
-                    placeholder="Select the Field do you want match"
-                    options={fieldOptions}
-                    onChange={event => onChangeMatchField(event.target.value)}
-                    value={matchField}
+                    value={actionOption}
+                    label="Merge exist records"
+                    description="Allows you to create only unique records avoiding duplications with existing records."
                 />
-            </RenderIf>
-        </StyledContainer>
+            ),
+            name: 'merge-records',
+            id: 'merge-records',
+        },
+    ];
+
+    return (
+        <>
+            <StyledContainer>
+                <StyledRadioGroup
+                    onChange={onChange}
+                    value={actionOption}
+                    options={options}
+                    id="radio-group-component-1"
```

Comment:
what is this ID used for? remember that ids needs to be unique, first we need to avoid using id at all inside components but if it is completely necessary then we need to generate one instead of hard coded